lua/ttt2/libraries/weaponrenderer.lua

```diff
@@ -131,6 +131,8 @@ function weaponrenderer.CreateModel(wep, modelData)
     then
         -- note on the rendergroup: This rendergroup is not listed in the GMod wiki; however all
         -- SWEP construction kit weapons use this rendergroup - and it works
+        -- FIXME: AFAIK this 'rendergroup' evaluates to `nil` so we would be better off simply
+        -- dropping this. Alternatively we could look into setting a proper RENDERGROUP instead
```

Comment:
huh, it does work that way - that's all I can say here

Reply:
i mean if it works with being `nil` we should simply remove this to prevent further confusion, right?

Reply:
probably yes?

Reply:
removed in [7ed4347](https://github.com/TTT-2/TTT2/pull/1574/commits/7ed4347fafa536cdf05d130afe3b4dabd48c199e)